backend/src/agents/config/agents.yaml

```
@@ -0,0 +1,52 @@
+    
+    ERROR HANDLING:
+    • Tool returns None → Consider SUCCESS unless explicit error message
+    • Tool output empty → Consider SUCCESS for actions like screenshot, navigation
+    • Don't retry identical tool calls → Move to next step or report completion
+    
+    COMPLETION INDICATORS:
+    When task is complete, include these phrases in your response:
+    • "user request fulfilled"
+    • "task completed"
+    • "automation complete"
+      
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)